src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -417,5 +417,16 @@ def tab_link(label, path, active = false)
 
     link_to(label, path, class: html_class)
   end
+
+  def webui2_user_or_group_image_tag(user_or_group:, css_class: nil, alt: nil, title: nil, size: 500)
+    user_or_group.is_a?(Group) ? (alt ||= (user_or_group ? "#{user_or_group.title}'s avatar" : 'Avatar')) :
```

Comment:
I would split this up to avoid duplication:
```ruby
name = user_or_group.is_a?(Group) ? user_or_group.title : user_or_group.name
alt ||= (user_or_group ? "#{name}'s avatar" : 'Avatar')
```